src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -678,7 +686,8 @@ public void SaveSettings()
                 FileConflictAction = _detailModel.Options.SelectedFileConflictAction.Action,
                 IncludePrerelease = _topPanel.CheckboxPrerelease.IsChecked == true,
                 SelectedFilter = _topPanel.Filter,
-                OptionsExpanded = _packageDetail._optionsControl.IsExpanded
+                OptionsExpanded = _packageDetail._optionsControl.IsExpanded,
+                SelectedPackageMetadataTab = PackageDetailsTabViewModel.ConvertFromTabType(_packageDetail._packageDetailsTabControl.PackageDetailsTabViewModel.SelectedTab)
```

Comment:
I believe I mentioned this in previous PR reviews, but it's not clear why the PackageManagerControl needs to be directly involved with the details pane's tabs. There are multiple models and ViewModels between this control and the tab controls. Could you explain design at a high level so I know the reasoning behind this?

Reply:
This is where the user settings are being saved. I need to get the current selected metadata tab in order to preserve the selected tab when the PM UI is opened again.